src/test/java/hudson/plugins/git/extensions/GitSCMExtensionTest.java

```diff
@@ -24,6 +24,9 @@
 
 	protected TaskListener listener;
 
+    @ClassRule
```

Comment:
Could this block (optionally) be indented to match the preceding and the following block?

My apologies that the code formatting in the plugin is already a horror.  It won't be less horrible by making this consistent with the preceding and following lines, but if you're willing, it would avoid adding one more formatting inconsistency.

Reply:
The rest of the file is using tabs. This I apparently pasted with spaces (a 4-space indent), which I did not notice since I usually display tabs as four spaces. I can switch it to tabs to make it consistent.